diff -ur otrs-3.0.5/bin/Cron.sh otrs-3.0.5.new//bin/Cron.sh
--- otrs-3.0.5/bin/Cron.sh	2009-02-26 12:01:01.000000000 +0100
+++ otrs-3.0.5.new//bin/Cron.sh	2011-11-03 11:25:00.450333939 +0100
@@ -45,7 +45,7 @@
 OTRS_HOME="`pwd`"
 cd -
 
-#OTRS_ROOT=/opt/OpenTRS
+#OTRS_ROOT=/usr/share/otrs
 if test -e $OTRS_HOME/var/cron; then
     OTRS_ROOT=$OTRS_HOME
 else
diff -ur otrs-3.0.5/bin/otrs.SetPermissions.pl otrs-3.0.5.new//bin/otrs.SetPermissions.pl
--- otrs-3.0.5/bin/otrs.SetPermissions.pl	2010-12-17 08:35:44.000000000 +0100
+++ otrs-3.0.5.new//bin/otrs.SetPermissions.pl	2011-11-03 11:25:31.843666857 +0100
@@ -76,7 +76,7 @@
     [--not-root]
     <OTRS_HOME>
 
-Try: otrs.SetPermissions.pl /opt/otrs
+Try: otrs.SetPermissions.pl /usr/share/otrs
 EOF
 
     if ( $#ARGV < 0 ) {
diff -ur otrs-3.0.5/Kernel/Config/Defaults.pm otrs-3.0.5.new//Kernel/Config/Defaults.pm
--- otrs-3.0.5/Kernel/Config/Defaults.pm	2011-01-06 10:54:05.000000000 +0100
+++ otrs-3.0.5.new//Kernel/Config/Defaults.pm	2011-11-03 11:26:15.786999604 +0100
@@ -295,7 +295,7 @@
 #    $Self->{'LogModule::SysLog::Charset'} = 'utf-8';
 
     # param for LogModule Kernel::System::Log::File (required!)
-    $Self->{'LogModule::LogFile'} = '/tmp/otrs.log';
+    $Self->{'LogModule::LogFile'} = '/var/log/otrs.log';
 
     # param if the date (yyyy-mm) should be added as suffix to
     # logfile [0|1]
@@ -827,7 +827,7 @@
     # directories                                         #
     # --------------------------------------------------- #
     # root directory
-    $Self->{Home} = '/opt/otrs';
+    $Self->{Home} = '/usr/share/otrs';
 
     # tmp dir
     $Self->{TempDir} = '<OTRS_CONFIG_Home>/var/tmp';
@@ -980,7 +980,7 @@
     # --------------------------------------------------- #
     $Self->{PGP}          = 0;
     $Self->{'PGP::Bin'}     = '/usr/bin/gpg';
-    $Self->{'PGP::Options'} = '--homedir /opt/otrs/.gnupg/ --batch --no-tty --yes';
+    $Self->{'PGP::Options'} = '--homedir /usr/share/otrs/.gnupg/ --batch --no-tty --yes';
 
 #    $Self->{'PGP::Options'} = '--batch --no-tty --yes';
 #    $Self->{'PGP::Key::Password'}->{'D2DF79FA'} = 1234;
diff -ur otrs-3.0.5/Kernel/Config/Files/Framework.xml otrs-3.0.5.new//Kernel/Config/Files/Framework.xml
--- otrs-3.0.5/Kernel/Config/Files/Framework.xml	2011-01-12 23:57:22.000000000 +0100
+++ otrs-3.0.5.new//Kernel/Config/Files/Framework.xml	2011-11-03 11:26:57.030331955 +0100
@@ -504,7 +504,7 @@
         <Group>Framework</Group>
         <SubGroup>Core::Log</SubGroup>
         <Setting>
-            <String Regex="">/tmp/otrs.log</String>
+            <String Regex="">/var/log/otrs.log</String>
         </Setting>
     </ConfigItem>
     <ConfigItem Name="LogModule::LogFile::Date" Required="1" Valid="1">
@@ -2781,7 +2781,7 @@
         <Group>Framework</Group>
         <SubGroup>Crypt::PGP</SubGroup>
         <Setting>
-            <String Regex="">--homedir /opt/otrs/.gnupg/ --batch --no-tty --yes</String>
+            <String Regex="">--homedir /usr/share/otrs/.gnupg/ --batch --no-tty --yes</String>
         </Setting>
     </ConfigItem>
     <ConfigItem Name="PGP::Key::Password" Required="1" Valid="1">
diff -ur otrs-3.0.5/Kernel/Config.pm.dist otrs-3.0.5.new//Kernel/Config.pm.dist
--- otrs-3.0.5/Kernel/Config.pm.dist	2010-01-13 23:25:00.000000000 +0100
+++ otrs-3.0.5.new//Kernel/Config.pm.dist	2011-11-03 11:27:15.280332150 +0100
@@ -59,7 +59,7 @@
     # ---------------------------------------------------- #
     # fs root directory
     # ---------------------------------------------------- #
-    $Self->{Home} = '/opt/otrs';
+    $Self->{Home} = '/usr/share/otrs';
 
     # ---------------------------------------------------- #
     # insert your own config settings "here"               #
diff -ur otrs-3.0.5/scripts/apache2-httpd.include.conf otrs-3.0.5.new//scripts/apache2-httpd.include.conf
--- otrs-3.0.5/scripts/apache2-httpd.include.conf	2010-10-11 18:16:11.000000000 +0200
+++ otrs-3.0.5.new//scripts/apache2-httpd.include.conf	2011-11-03 11:18:32.017005757 +0100
@@ -4,14 +4,14 @@
 # --
 
 # agent, admin and customer frontend
-ScriptAlias /otrs/ "/opt/otrs/bin/cgi-bin/"
-Alias /otrs-web/ "/opt/otrs/var/httpd/htdocs/"
+ScriptAlias /otrs/ "/usr/share/otrs/bin/cgi-bin/"
+Alias /otrs-web/ "/usr/share/otrs/var/httpd/htdocs/"
 
 # if mod_perl is used
 <IfModule mod_perl.c>
 
     # load all otrs modules
-    Perlrequire /opt/otrs/scripts/apache2-perl-startup.pl
+    Perlrequire /usr/share/otrs/scripts/apache2-perl-startup.pl
 
     # Apache::Reload - Reload Perl Modules when Changed on Disk
     PerlModule Apache2::Reload
@@ -34,26 +34,26 @@
 </IfModule>
 
 # directory settings
-<Directory "/opt/otrs/bin/cgi-bin/">
+<Directory "/usr/share/otrs/bin/cgi-bin/">
     AllowOverride None
     Options +ExecCGI -Includes
     Order allow,deny
     Allow from all
 </Directory>
-<Directory "/opt/otrs/var/httpd/htdocs/">
+<Directory "/usr/share/otrs/var/httpd/htdocs/">
     AllowOverride None
     Order allow,deny
     Allow from all
 </Directory>
 
 <IfModule mod_headers.c>
-    <Directory "/opt/otrs/var/httpd/htdocs/skins/*/*/css-cache">
+    <Directory "/usr/share/otrs/var/httpd/htdocs/skins/*/*/css-cache">
         <FilesMatch "\.(css|CSS)$">
             Header set Cache-Control "max-age=2592000 must-revalidate"
         </FilesMatch>
     </Directory>
 
-    <Directory "/opt/otrs/var/httpd/htdocs/js_new/js-cache">
+    <Directory "/usr/share/otrs/var/httpd/htdocs/js_new/js-cache">
         <FilesMatch "\.(js|JS)$">
             Header set Cache-Control "max-age=2592000 must-revalidate"
         </FilesMatch>
diff -ur otrs-3.0.5/scripts/apache2-perl-startup.pl otrs-3.0.5.new//scripts/apache2-perl-startup.pl
--- otrs-3.0.5/scripts/apache2-perl-startup.pl	2010-09-03 09:10:35.000000000 +0200
+++ otrs-3.0.5.new//scripts/apache2-perl-startup.pl	2011-11-03 11:29:49.820331074 +0100
@@ -35,9 +35,9 @@
 use ModPerl::Util;
 
 # set otrs lib path!
-use lib "/opt/otrs/";
-use lib "/opt/otrs/Kernel/cpan-lib";
-use lib "/opt/otrs/Custom";
+use lib "/usr/share/otrs/";
+use lib "/usr/share/otrs/Kernel/cpan-lib";
+use lib "/usr/share/otrs/Custom";
 
 # pull in things we will use in most requests so it is read and compiled
 # exactly once
