--- ./bin/Cron.sh.org	2009-02-11 12:39:13.747483330 +0100
+++ ./bin/Cron.sh	2009-02-11 12:40:05.377477480 +0100
@@ -44,12 +44,12 @@ cd "`dirname $0`/../"
 OTRS_HOME="`pwd`"
 cd -
 
-#OTRS_ROOT=/opt/OpenTRS
+#OTRS_ROOT=/usr/share/otrs
 if test -e $OTRS_HOME/var/cron; then
     OTRS_ROOT=$OTRS_HOME
 else
     echo "No cronjobs in $OTRS_HOME/var/cron found!";
-    echo " * Check the \$HOME (/etc/passwd) of the OTRS user. It must be the root dir of your OTRS system (e. g. /opt/otrs). ";
+    echo " * Check the \$HOME (/etc/passwd) of the OTRS user. It must be the root dir of your OTRS system (e. g. /usr/share/otrs). ";
     exit 5;
 fi
 
--- ./bin/SetPermissions.sh.org	2009-02-11 12:40:15.234559506 +0100
+++ ./bin/SetPermissions.sh	2009-02-11 12:40:40.067473048 +0100
@@ -28,7 +28,7 @@ if ! test $1 || ! test $2 || ! test $3; 
     echo ""
     echo "Usage: SetPermissions.sh <OTRS_HOME> <OTRS_USER> <WEBSERVER_USER> [OTRS_GROUP] [WEB_GROUP]"
     echo ""
-    echo "  Try: SetPermissions.sh /opt/otrs otrs wwwrun"
+    echo "  Try: SetPermissions.sh /usr/share/otrs otrs wwwrun"
     echo ""
     exit 1;
 else
--- ./CHANGES.org	2009-02-11 12:40:52.087608830 +0100
+++ ./CHANGES	2009-02-11 12:41:43.544347438 +0100
@@ -1582,7 +1582,7 @@
  - (2006/09/27) fixed bug #1358 - Customer History > All customer tickets.
     returns an incorrect number of tickets
  - (2006/09/27) fixed bug #1293 - Can't use an undefined value as a HASH
-    reference at /opt/otrs/bin/cgi-bin//../../Kernel/Modules/AgentTicketBulk.pm
+    reference at /usr/share/otrs/bin/cgi-bin//../../Kernel/Modules/AgentTicketBulk.pm
     line 194
  - (2006/09/27) fixed bug #1356 - mssql driver bug - Stats module of OTRS 2.1
     beta 2 - "Cant use string ("0")".
@@ -3146,7 +3146,7 @@
     (e. g. Kernel/Language/bg.pm). Updated docu.
  - (2002/11/21) moved var/cron/* to var/cron/*.dist (.dist is not used) to make
     updates easier! Thanks to Bryan Fullerton!
- - (2002/11/15) moved %doc/install/* to /opt/OpenTRS/install in RPM-specs.
+ - (2002/11/15) moved %doc/install/* to /usr/share/OpenTRS/install in RPM-specs.
     because the web-installer needs this stuff in this location. %doc isn't
     consistent on different linux distributions!
  - (2002/11/15) fixed bug [48], custom modules don't work/load -
@@ -3319,7 +3319,7 @@
  - fixed bug[12] fetchmail lock problem - http://bugs.otrs.org/show_bug.cgi?id=12
  - fixed bug[11] typos - http://bugs.otrs.org/show_bug.cgi?id=11
  - fixed bug[10] user_preferences table  - http://bugs.otrs.org/show_bug.cgi?id=10
- - fixed bug[9] LoopProtection!!! Can't open'/opt/OpenTRS/var/log/LoopProtection-xyz.log':
+ - fixed bug[9] LoopProtection!!! Can't open'/usr/share/OpenTRS/var/log/LoopProtection-xyz.log':
     No such file or directory! - http://bugs.otrs.org/show_bug.cgi?id=9
  - fixed HTML table bug in AdminArea::Queue (just with Netscape)
  - fixed SQL table preferences bug (use script/DBUpdate.mysql.sql)
--- ./scripts/apache2-perl-startup.pl.org	2008-03-28 12:40:15.000000000 +0100
+++ ./scripts/apache2-perl-startup.pl	2009-02-11 13:05:06.537673495 +0100
@@ -8,8 +8,8 @@ use warnings;
 $ENV{MOD_PERL} =~ /mod_perl/ or die "MOD_PERL not used!";
 
 # set otrs lib path!
-use lib "/opt/otrs/";
-use lib "/opt/otrs/Kernel/cpan-lib";
+use lib "/usr/share/otrs/";
+use lib "/usr/share/otrs/Kernel/cpan-lib";
 
 # pull in things we will use in most requests so it is read and compiled
 # exactly once
