--- otrs/Kernel/Config/Defaults.pm.orig	2004-09-16 08:06:01.000000000 +0000
+++ otrs/Kernel/Config/Defaults.pm	2004-10-14 06:27:08.016383496 +0000
@@ -549,7 +549,7 @@
 #    $Self->{'LogModule::SysLog::Charset'} = 'utf-8';
 
     # param for LogModule Kernel::System::Log::File (required!)
-    $Self->{'LogModule::LogFile'} = '/tmp/otrs.log';
+    $Self->{'LogModule::LogFile'} = '/var/log/otrs/otrs.log';
 
     # param if the date (yyyy-mm) should be added as suffix to
     # logfile [0|1]
@@ -610,7 +610,7 @@
     # root directory
     $Self->{Home} = '/opt/otrs';
     # counter log
-    $Self->{CounterLog} = '<OTRS_CONFIG_Home>/var/log/TicketCounter.log';
+    $Self->{CounterLog} = '/var/log/otrs/TicketCounter.log';
     # article fs dir
     $Self->{ArticleDir} = '<OTRS_CONFIG_Home>/var/article';
     # stats dir
