--- otrs-3.2.1/bin/otrs.SetPermissions.pl	2013-01-24 11:21:14.000000000 +0100
+++ otrs-3.2.1.new/bin/otrs.SetPermissions.pl	2013-01-29 12:39:39.175799680 +0100
@@ -82,7 +82,7 @@
     [--not-root]
     <OTRS_HOME>
 
-Try: otrs.SetPermissions.pl /opt/otrs
+Try: otrs.SetPermissions.pl /usr/share/otrs
 EOF
 
     if ( $#ARGV < 0 ) {
--- otrs-3.2.1/Kernel/Config/Defaults.pm	2013-01-17 10:06:52.000000000 +0100
+++ otrs-3.2.1.new/Kernel/Config/Defaults.pm	2013-01-29 12:40:37.118464479 +0100
@@ -297,7 +297,7 @@
 #    $Self->{'LogModule::SysLog::Charset'} = 'utf-8';
 
     # param for LogModule Kernel::System::Log::File (required!)
-    $Self->{'LogModule::LogFile'} = '/tmp/otrs.log';
+    $Self->{'LogModule::LogFile'} = '/var/log/otrs.log';
 
     # param if the date (yyyy-mm) should be added as suffix to
     # logfile [0|1]
@@ -824,7 +824,7 @@
     # directories                                         #
     # --------------------------------------------------- #
     # root directory
-    $Self->{Home} = '/opt/otrs';
+    $Self->{Home} = '/usr/share/otrs';
 
     # tmp dir
     $Self->{TempDir} = '<OTRS_CONFIG_Home>/var/tmp';
@@ -965,7 +965,7 @@
     # --------------------------------------------------- #
     $Self->{PGP}          = 0;
     $Self->{'PGP::Bin'}     = '/usr/bin/gpg';
-    $Self->{'PGP::Options'} = '--homedir /opt/otrs/.gnupg/ --batch --no-tty --yes';
+    $Self->{'PGP::Options'} = '--homedir /usr/share/otrs/.gnupg/ --batch --no-tty --yes';
 
 #    $Self->{'PGP::Options'} = '--batch --no-tty --yes';
 #    $Self->{'PGP::Key::Password'}->{'D2DF79FA'} = 1234;
--- otrs-3.2.1/Kernel/Config/Files/Framework.xml	2013-01-03 10:12:36.000000000 +0100
+++ otrs-3.2.1.new/Kernel/Config/Files/Framework.xml	2013-01-29 12:41:09.689962451 +0100
@@ -493,7 +493,7 @@
         <Group>Framework</Group>
         <SubGroup>Core::Log</SubGroup>
         <Setting>
-            <String Regex="">/tmp/otrs.log</String>
+		<String Regex="">/usr/share/otrs.log</String>
         </Setting>
     </ConfigItem>
     <ConfigItem Name="LogModule::LogFile::Date" Required="1" Valid="1">
@@ -2834,7 +2834,7 @@
         <Group>Framework</Group>
         <SubGroup>Crypt::PGP</SubGroup>
         <Setting>
-            <String Regex="">--homedir /opt/otrs/.gnupg/ --batch --no-tty --yes</String>
+		<String Regex="">--homedir /usr/share/otrs/.gnupg/ --batch --no-tty --yes</String>
         </Setting>
     </ConfigItem>
     <ConfigItem Name="PGP::Key::Password" Required="1" Valid="1">
--- otrs-3.2.1/Kernel/Config.pm.dist	2012-11-22 12:43:05.000000000 +0100
+++ otrs-3.2.1.new/Kernel/Config.pm.dist	2013-01-29 12:41:32.054324327 +0100
@@ -63,7 +63,7 @@
     # ---------------------------------------------------- #
     # fs root directory
     # ---------------------------------------------------- #
-    $Self->{Home} = '/opt/otrs';
+    $Self->{Home} = '/usr/share/otrs';
 
     # ---------------------------------------------------- #
     # insert your own config settings "here"               #
--- otrs-3.2.1/scripts/apache2-httpd.include.conf	2013-01-17 10:06:55.000000000 +0100
+++ otrs-3.2.1.new/scripts/apache2-httpd.include.conf	2013-01-29 12:42:41.307509300 +0100
@@ -3,8 +3,8 @@
 # $Id: apache2-httpd.include.conf,v 1.27 2013/01/04 13:03:06 mg Exp $
 # --
 
-ScriptAlias /otrs/ "/opt/otrs/bin/cgi-bin/"
-Alias /otrs-web/ "/opt/otrs/var/httpd/htdocs/"
+ScriptAlias /otrs/ "/usr/share/otrs/bin/cgi-bin/"
+Alias /otrs-web/ "/usr/share/otrs/var/httpd/htdocs/"
 
 # activate this if you are using an Oracle database
 #SetEnv ORACLE_HOME /path/to/your/oracle/
@@ -15,7 +15,7 @@
 <IfModule mod_perl.c>
 
     # Setup environment and preload modules
-    Perlrequire /opt/otrs/scripts/apache2-perl-startup.pl
+    Perlrequire /usr/share/otrs/scripts/apache2-perl-startup.pl
 
     # Reload Perl modules when changed on disk
     PerlModule Apache2::Reload
@@ -41,7 +41,7 @@
 
 </IfModule>
 
-<Directory "/opt/otrs/bin/cgi-bin/">
+<Directory "/usr/share/otrs/bin/cgi-bin/">
     AllowOverride None
     Options +ExecCGI -Includes
     Order allow,deny
@@ -52,7 +52,7 @@
     </IfModule>
 </Directory>
 
-<Directory "/opt/otrs/var/httpd/htdocs/">
+<Directory "/usr/share/otrs/var/httpd/htdocs/">
     AllowOverride None
     Order allow,deny
     Allow from all
@@ -63,13 +63,13 @@
 </Directory>
 
 <IfModule mod_headers.c>
-    <Directory "/opt/otrs/var/httpd/htdocs/skins/*/*/css-cache">
+    <Directory "/usr/share/otrs/var/httpd/htdocs/skins/*/*/css-cache">
         <FilesMatch "\.(css|CSS)$">
             Header set Cache-Control "max-age=2592000 must-revalidate"
         </FilesMatch>
     </Directory>
 
-    <Directory "/opt/otrs/var/httpd/htdocs/js/js-cache">
+    <Directory "/usr/share/otrs/var/httpd/htdocs/js/js-cache">
         <FilesMatch "\.(js|JS)$">
             Header set Cache-Control "max-age=2592000 must-revalidate"
         </FilesMatch>
--- otrs-3.2.1/scripts/apache2-perl-startup.pl	2013-01-24 11:21:14.000000000 +0100
+++ otrs-3.2.1.new/scripts/apache2-perl-startup.pl	2013-01-29 12:43:07.268703262 +0100
@@ -35,15 +35,15 @@
     # set $0 to index.pl because this is broken in mod_perl context;
     # apart from that, on Fedora, $0 is not a path which would break OTRS.
     # see bug # 8533
-    $0 = '/opt/otrs/bin/cgi-bin/index.pl';
+    $0 = '/usr/share/otrs/bin/cgi-bin/index.pl';
 }
 
 use Apache2::RequestRec;
 use ModPerl::Util;
 
-use lib "/opt/otrs/";
-use lib "/opt/otrs/Kernel/cpan-lib";
-use lib "/opt/otrs/Custom";
+use lib "/usr/share/otrs/";
+use lib "/usr/share/otrs/Kernel/cpan-lib";
+use lib "/usr/share/otrs/Custom";
 
 # Preload frequently used modules to speed up client spawning.
 use CGI ();
--- otrs-3.2.1/scripts/redhat-rcotrs-config	2012-10-11 09:11:09.000000000 +0200
+++ otrs-3.2.1.new/scripts/redhat-rcotrs-config	2013-01-29 12:43:27.239621730 +0100
@@ -12,7 +12,7 @@
 # 
 # OTRS home (root) directory
 #
-OTRS_ROOT=/opt/otrs
+OTRS_ROOT=/usr/share/otrs
 
 # if one of this is false, it will not be checked at the startup!
 # Note: (may be the database isn't on the same host! --> DB_RUNNING=0) 
--- otrs-3.2.1/bin/Cron.sh	2012-11-22 12:43:09.000000000 +0100
+++ otrs-3.2.1.new/bin/Cron.sh	2013-01-31 09:29:55.308734799 +0100
@@ -45,16 +45,16 @@
 OTRS_HOME="`pwd`"
 cd -
 
-#OTRS_ROOT=/opt/OpenTRS
-if test -e $OTRS_HOME/var/cron; then
+#OTRS_ROOT=/usr/share/otrs
+if test -e /etc/cron.d; then
     OTRS_ROOT=$OTRS_HOME
 else
-    echo "No cronjobs in $OTRS_HOME/var/cron found!";
+    echo "No cronjobs in /etc/cron.d found!";
     echo " * Check the \$HOME (/etc/passwd) of the OTRS user. It must be the root dir of your OTRS system (e. g. /opt/otrs). ";
     exit 5;
 fi
 
-CRON_DIR=$OTRS_ROOT/var/cron
+CRON_DIR=/etc/cron.d
 CRON_TMP_FILE=$OTRS_ROOT/var/tmp/otrs-cron-tmp.$$
 
 echo "Cron.sh - start/stop OTRS cronjobs - <\$Revision: 1.22 $> "
