--- otrs/scripts/redhat-rcotrs-config.fix	Mon Mar 15 11:13:09 2004
+++ otrs/scripts/redhat-rcotrs-config	Mon Mar 15 11:24:00 2004
@@ -12,7 +12,7 @@
 # 
 # OTRS home (root) directory
 #
-OTRS_ROOT=/opt/otrs
+OTRS_ROOT=/usr/share/otrs
 
 # if one of this is false, it will not be checked at the startup!
 # Note: (may be the database isn't on the same host! --> DB_RUNNING=0) 
@@ -27,7 +27,8 @@
 #
 OTRS_USED_DB=mysqld
 #OTRS_USED_DB_TEST="ps --pid $(cat /var/lib/mysql/$HOST.pid)"
-OTRS_USED_DB_TEST="/sbin/service mysqld status | grep 'is running'"
+#OTRS_USED_DB_TEST="/sbin/service mysqld status | grep 'is running'"
+OTRS_USED_DB_TEST="/etc/rc.d/init.d/mysql status | grep 'pid'"
 # --- The Wonderfull redhat's mysql init script does not have a status check.. sucks
 #OTRS_USED_DB=PostgreSQL
 #OTRS_USED_DB_RCSCRIPT="service postgresql status| grep 'is running' > /dev/null 2>&1"
@@ -37,7 +38,8 @@
 # Used webserver [apache]
 #
 OTRS_USED_WEBSERVER=httpd   #Redhat's default httpd is apache..
-OTRS_USED_WEBSERVER_TEST="/sbin/service httpd status | grep 'is running'"
+#OTRS_USED_WEBSERVER_TEST="/sbin/service httpd status | grep 'is running'"
+OTRS_USED_WEBSERVER_TEST="/etc/rc.d/init.d/httpd status | grep 'pid'"
 
 #
 # cron stuff
