diff -ur otrs-3.3.5/bin/Cron.sh otrs-3.3.5.new/bin/Cron.sh
--- otrs-3.3.5/bin/Cron.sh	2014-02-20 09:22:07.000000000 +0100
+++ otrs-3.3.5.new/bin/Cron.sh	2014-02-28 13:44:49.814960681 +0100
@@ -43,16 +43,16 @@
 OTRS_HOME="`pwd`"
 cd -
 
-#OTRS_ROOT=/opt/otrs
-if test -e $OTRS_HOME/var/cron; then
+#OTRS_ROOT=/usr/share/otrs
+if test -e /etc/cron.d; then
     OTRS_ROOT=$OTRS_HOME
 else
-    echo "No cronjobs in $OTRS_HOME/var/cron found!";
-    echo " * Check the \$HOME (/etc/passwd) of the OTRS user. It must be the root dir of your OTRS system (e. g. /opt/otrs). ";
+    echo "No cronjobs in /etc/cron.d found!";
+    echo " * Check the \$HOME (/etc/passwd) of the OTRS user. It must be the root dir of your OTRS system (e. g. /usr/share/otrs). ";
     exit 5;
 fi
 
-CRON_DIR=$OTRS_ROOT/var/cron
+CRON_DIR=/etc/cron.d
 CRON_TMP_FILE=$OTRS_ROOT/var/tmp/otrs-cron-tmp.$$
 
 echo "Cron.sh - start/stop OTRS cronjobs"
Tylko w otrs-3.3.5.new/bin: Cron.sh~
diff -ur otrs-3.3.5/bin/otrs.SetPermissions.pl otrs-3.3.5.new/bin/otrs.SetPermissions.pl
--- otrs-3.3.5/bin/otrs.SetPermissions.pl	2014-02-20 09:22:07.000000000 +0100
+++ otrs-3.3.5.new/bin/otrs.SetPermissions.pl	2014-02-28 13:46:22.849207187 +0100
@@ -77,7 +77,7 @@
     [--not-root]
     <OTRS_HOME>
 
-Try: otrs.SetPermissions.pl /opt/otrs
+Try: otrs.SetPermissions.pl /usr/share/otrs
 EOF
 
     if ( $#ARGV < 0 ) {
Tylko w otrs-3.3.5.new/bin: otrs.SetPermissions.pl~
diff -ur otrs-3.3.5/Kernel/Config/Defaults.pm otrs-3.3.5.new/Kernel/Config/Defaults.pm
--- otrs-3.3.5/Kernel/Config/Defaults.pm	2014-02-20 09:22:11.000000000 +0100
+++ otrs-3.3.5.new/Kernel/Config/Defaults.pm	2014-02-28 13:47:21.181869809 +0100
@@ -292,7 +292,7 @@
 #    $Self->{'LogModule::SysLog::Charset'} = 'utf-8';
 
     # param for LogModule Kernel::System::Log::File (required!)
-    $Self->{'LogModule::LogFile'} = '/tmp/otrs.log';
+    $Self->{'LogModule::LogFile'} = '/var/log/otrs/otrs.log';
 
     # param if the date (yyyy-mm) should be added as suffix to
     # logfile [0|1]
@@ -803,7 +803,7 @@
     # directories                                         #
     # --------------------------------------------------- #
     # root directory
-    $Self->{Home} = '/opt/otrs';
+    $Self->{Home} = '/usr/share/otrs';
 
     # tmp dir
     $Self->{TempDir} = '<OTRS_CONFIG_Home>/var/tmp';
@@ -966,7 +966,7 @@
     # --------------------------------------------------- #
     $Self->{PGP}          = 0;
     $Self->{'PGP::Bin'}     = '/usr/bin/gpg';
-    $Self->{'PGP::Options'} = '--homedir /opt/otrs/.gnupg/ --batch --no-tty --yes';
+    $Self->{'PGP::Options'} = '--homedir /usr/share/otrs/.gnupg/ --batch --no-tty --yes';
 
 #    $Self->{'PGP::Options'} = '--batch --no-tty --yes';
 #    $Self->{'PGP::Key::Password'}->{'D2DF79FA'} = 1234;
Tylko w otrs-3.3.5.new/Kernel/Config: Defaults.pm~
diff -ur otrs-3.3.5/Kernel/Config/Files/Framework.xml otrs-3.3.5.new/Kernel/Config/Files/Framework.xml
--- otrs-3.3.5/Kernel/Config/Files/Framework.xml	2014-02-20 09:22:11.000000000 +0100
+++ otrs-3.3.5.new/Kernel/Config/Files/Framework.xml	2014-02-28 13:48:17.254429304 +0100
@@ -507,7 +507,7 @@
         <Group>Framework</Group>
         <SubGroup>Core::Log</SubGroup>
         <Setting>
-            <String Regex="">/tmp/otrs.log</String>
+		<String Regex="">/var/log/otrs/otrs.log</String>
         </Setting>
     </ConfigItem>
     <ConfigItem Name="LogModule::LogFile::Date" Required="1" Valid="1">
@@ -2867,7 +2867,7 @@
         <Group>Framework</Group>
         <SubGroup>Crypt::PGP</SubGroup>
         <Setting>
-            <String Regex="">--homedir /opt/otrs/.gnupg/ --batch --no-tty --yes</String>
+		<String Regex="">--homedir /usr/share/otrs/.gnupg/ --batch --no-tty --yes</String>
         </Setting>
     </ConfigItem>
     <ConfigItem Name="PGP::Key::Password" Required="1" Valid="1">
Tylko w otrs-3.3.5.new/Kernel/Config/Files: Framework.xml~
diff -ur otrs-3.3.5/Kernel/Config/Files/Ticket.xml otrs-3.3.5.new/Kernel/Config/Files/Ticket.xml
--- otrs-3.3.5/Kernel/Config/Files/Ticket.xml	2014-02-20 09:22:11.000000000 +0100
+++ otrs-3.3.5.new/Kernel/Config/Files/Ticket.xml	2014-02-28 13:48:32.905143705 +0100
@@ -225,7 +225,7 @@
         <Group>Ticket</Group>
         <SubGroup>Core::Ticket</SubGroup>
         <Setting>
-            <String Regex="">&lt;OTRS_CONFIG_Home&gt;/var/log/TicketCounter.log</String>
+		<String Regex="">&lt;OTRS_CONFIG_Home&gt;/var/log/otrs/TicketCounter.log</String>
         </Setting>
     </ConfigItem>
     <ConfigItem Name="Ticket::IndexModule" Required="1" Valid="1">
Tylko w otrs-3.3.5.new/Kernel/Config/Files: Ticket.xml~
diff -ur otrs-3.3.5/Kernel/Config.pm.dist otrs-3.3.5.new/Kernel/Config.pm.dist
--- otrs-3.3.5/Kernel/Config.pm.dist	2014-02-20 09:22:06.000000000 +0100
+++ otrs-3.3.5.new/Kernel/Config.pm.dist	2014-02-28 13:48:55.052821342 +0100
@@ -67,7 +67,7 @@
     # ---------------------------------------------------- #
     # fs root directory
     # ---------------------------------------------------- #
-    $Self->{Home} = '/opt/otrs';
+    $Self->{Home} = '/usr/share/otrs';
 
     # ---------------------------------------------------- #
     # insert your own config settings "here"               #
Tylko w otrs-3.3.5.new/Kernel: Config.pm.dist~
diff -ur otrs-3.3.5/scripts/apache2-httpd.include.conf otrs-3.3.5.new/scripts/apache2-httpd.include.conf
--- otrs-3.3.5/scripts/apache2-httpd.include.conf	2014-02-20 09:22:07.000000000 +0100
+++ otrs-3.3.5.new/scripts/apache2-httpd.include.conf	2014-02-28 13:50:28.093735087 +0100
@@ -2,8 +2,8 @@
 # added for OTRS (http://otrs.org/)
 # --
 
-ScriptAlias /otrs/ "/opt/otrs/bin/cgi-bin/"
-Alias /otrs-web/ "/opt/otrs/var/httpd/htdocs/"
+ScriptAlias /otrs/ "/usr/share/otrs/bin/cgi-bin/"
+Alias /otrs-web/ "/usr/share/otrs/var/httpd/htdocs/"
 
 # activate this if you are using an Oracle database
 #SetEnv ORACLE_HOME /path/to/your/oracle/
@@ -14,7 +14,7 @@
 <IfModule mod_perl.c>
 
     # Setup environment and preload modules
-    Perlrequire /opt/otrs/scripts/apache2-perl-startup.pl
+    Perlrequire /usr/share/otrs/scripts/apache2-perl-startup.pl
 
     # Reload Perl modules when changed on disk
     PerlModule Apache2::Reload
@@ -52,7 +52,7 @@
 
 </IfModule>
 
-<Directory "/opt/otrs/bin/cgi-bin/">
+<Directory "/usr/share/otrs/bin/cgi-bin/">
     AllowOverride None
     Options +ExecCGI -Includes
 
@@ -75,7 +75,7 @@
     </IfModule>
 </Directory>
 
-<Directory "/opt/otrs/var/httpd/htdocs/">
+<Directory "/usr/share/otrs/var/httpd/htdocs/">
     AllowOverride None
 
     <IfModule mod_version.c>
@@ -107,28 +107,28 @@
 
 <IfModule mod_headers.c>
     # Cache css-cache for 30 days
-    <Directory "/opt/otrs/var/httpd/htdocs/skins/*/*/css-cache">
+    <Directory "/usr/share/otrs/var/httpd/htdocs/skins/*/*/css-cache">
         <FilesMatch "\.(css|CSS)$">
             Header set Cache-Control "max-age=2592000 must-revalidate"
         </FilesMatch>
     </Directory>
 
     # Cache css thirdparty for 4 hours, including icon fonts
-    <Directory "/opt/otrs/var/httpd/htdocs/skins/*/*/css/thirdparty">
+    <Directory "/usr/share/otrs/var/httpd/htdocs/skins/*/*/css/thirdparty">
         <FilesMatch "\.(css|CSS|woff|svg)$">
             Header set Cache-Control "max-age=14400 must-revalidate"
         </FilesMatch>
     </Directory>
 
     # Cache js-cache for 30 days
-    <Directory "/opt/otrs/var/httpd/htdocs/js/js-cache">
+    <Directory "/usr/share/otrs/var/httpd/htdocs/js/js-cache">
         <FilesMatch "\.(js|JS)$">
             Header set Cache-Control "max-age=2592000 must-revalidate"
         </FilesMatch>
     </Directory>
 
     # Cache js thirdparty for 4 hours
-    <Directory "/opt/otrs/var/httpd/htdocs/js/thirdparty/">
+    <Directory "/usr/share/otrs/var/httpd/htdocs/js/thirdparty/">
         <FilesMatch "\.(js|JS)$">
             Header set Cache-Control "max-age=14400 must-revalidate"
         </FilesMatch>
@@ -136,4 +136,4 @@
 </IfModule>
 
 # Limit the number of requests per child to avoid excessive memory usage
-MaxRequestsPerChild 4000
+MaxRequestsPerChild 4000`
Tylko w otrs-3.3.5.new/scripts: apache2-httpd.include.conf~
diff -ur otrs-3.3.5/scripts/apache2-perl-startup.pl otrs-3.3.5.new/scripts/apache2-perl-startup.pl
--- otrs-3.3.5/scripts/apache2-perl-startup.pl	2014-02-20 09:22:07.000000000 +0100
+++ otrs-3.3.5.new/scripts/apache2-perl-startup.pl	2014-02-28 13:51:10.749017805 +0100
@@ -35,16 +35,16 @@
     # on Fedora, $0 is not a path which would break OTRS.
     # see bug # 8533
     if ( !-e $0 ) {
-        $0 = '/opt/otrs/bin/cgi-bin/index.pl';
+        $0 = '/usr/share/otrs/bin/cgi-bin/index.pl';
     }
 }
 
 use Apache2::RequestRec;
 use ModPerl::Util;
 
-use lib "/opt/otrs/";
-use lib "/opt/otrs/Kernel/cpan-lib";
-use lib "/opt/otrs/Custom";
+use lib "/usr/share/otrs/";
+use lib "/usr/share/otrs/Kernel/cpan-lib";
+use lib "/usr/share/otrs/Custom";
 
 # Preload frequently used modules to speed up client spawning.
 use CGI ();
Tylko w otrs-3.3.5.new/scripts: apache2-perl-startup.pl~
diff -ur otrs-3.3.5/scripts/redhat-rcotrs-config otrs-3.3.5.new/scripts/redhat-rcotrs-config
--- otrs-3.3.5/scripts/redhat-rcotrs-config	2014-02-20 09:14:42.000000000 +0100
+++ otrs-3.3.5.new/scripts/redhat-rcotrs-config	2014-02-28 13:51:27.989807583 +0100
@@ -10,7 +10,7 @@
 # 
 # OTRS home (root) directory
 #
-OTRS_ROOT=/opt/otrs
+OTRS_ROOT=/usr/share/otrs
 
 # if one of this is false, it will not be checked at the startup!
 # Note: (may be the database isn't on the same host! --> DB_RUNNING=0) 
Tylko w otrs-3.3.5.new/scripts: redhat-rcotrs-config~
