diff -ur otrs-3.0.11/bin/Cron.sh otrs-3.0.11.new//bin/Cron.sh
--- otrs-3.0.11/bin/Cron.sh	2011-10-11 14:01:44.000000000 +0200
+++ otrs-3.0.11.new//bin/Cron.sh	2011-11-04 09:11:31.296872990 +0100
@@ -45,7 +45,7 @@
 OTRS_HOME="`pwd`"
 cd -
 
-#OTRS_ROOT=/opt/OpenTRS
+#OTRS_ROOT=/usr/share/otrs
 if test -e $OTRS_HOME/var/cron; then
     OTRS_ROOT=$OTRS_HOME
 else
diff -ur otrs-3.0.11/bin/otrs.SetPermissions.pl otrs-3.0.11.new//bin/otrs.SetPermissions.pl
--- otrs-3.0.11/bin/otrs.SetPermissions.pl	2011-10-11 14:01:45.000000000 +0200
+++ otrs-3.0.11.new//bin/otrs.SetPermissions.pl	2011-11-04 09:11:31.296872990 +0100
@@ -76,7 +76,7 @@
     [--not-root]
     <OTRS_HOME>
 
-Try: otrs.SetPermissions.pl /opt/otrs
+Try: otrs.SetPermissions.pl /usr/share/otrs
 EOF
 
     if ( $#ARGV < 0 ) {
diff -ur otrs-3.0.11/Kernel/Config/Defaults.pm otrs-3.0.11.new//Kernel/Config/Defaults.pm
--- otrs-3.0.11/Kernel/Config/Defaults.pm	2011-10-11 14:01:54.000000000 +0200
+++ otrs-3.0.11.new//Kernel/Config/Defaults.pm	2011-11-04 09:11:31.296872990 +0100
@@ -302,7 +302,7 @@
 #    $Self->{'LogModule::SysLog::Charset'} = 'utf-8';
 
     # param for LogModule Kernel::System::Log::File (required!)
-    $Self->{'LogModule::LogFile'} = '/tmp/otrs.log';
+    $Self->{'LogModule::LogFile'} = '/var/log/otrs.log';
 
     # param if the date (yyyy-mm) should be added as suffix to
     # logfile [0|1]
@@ -834,7 +834,7 @@
     # directories                                         #
     # --------------------------------------------------- #
     # root directory
-    $Self->{Home} = '/opt/otrs';
+    $Self->{Home} = '/usr/share/otrs';
 
     # tmp dir
     $Self->{TempDir} = '<OTRS_CONFIG_Home>/var/tmp';
@@ -987,7 +987,7 @@
     # --------------------------------------------------- #
     $Self->{PGP}          = 0;
     $Self->{'PGP::Bin'}     = '/usr/bin/gpg';
-    $Self->{'PGP::Options'} = '--homedir /opt/otrs/.gnupg/ --batch --no-tty --yes';
+    $Self->{'PGP::Options'} = '--homedir /usr/share/otrs/.gnupg/ --batch --no-tty --yes';
 
 #    $Self->{'PGP::Options'} = '--batch --no-tty --yes';
 #    $Self->{'PGP::Key::Password'}->{'D2DF79FA'} = 1234;
Tylko w otrs-3.0.11.new//Kernel/Config: Defaults.pm.orig
diff -ur otrs-3.0.11/Kernel/Config/Files/Framework.xml otrs-3.0.11.new//Kernel/Config/Files/Framework.xml
--- otrs-3.0.11/Kernel/Config/Files/Framework.xml	2011-10-11 14:01:54.000000000 +0200
+++ otrs-3.0.11.new//Kernel/Config/Files/Framework.xml	2011-11-04 09:11:31.300206326 +0100
@@ -499,7 +499,7 @@
         <Group>Framework</Group>
         <SubGroup>Core::Log</SubGroup>
         <Setting>
-            <String Regex="">/tmp/otrs.log</String>
+            <String Regex="">/var/log/otrs.log</String>
         </Setting>
     </ConfigItem>
     <ConfigItem Name="LogModule::LogFile::Date" Required="1" Valid="1">
@@ -2787,7 +2787,7 @@
         <Group>Framework</Group>
         <SubGroup>Crypt::PGP</SubGroup>
         <Setting>
-            <String Regex="">--homedir /opt/otrs/.gnupg/ --batch --no-tty --yes</String>
+            <String Regex="">--homedir /usr/share/otrs/.gnupg/ --batch --no-tty --yes</String>
         </Setting>
     </ConfigItem>
     <ConfigItem Name="PGP::Key::Password" Required="1" Valid="1">
Tylko w otrs-3.0.11.new//Kernel/Config/Files: Framework.xml.orig
diff -ur otrs-3.0.11/Kernel/Config.pm.dist otrs-3.0.11.new//Kernel/Config.pm.dist
--- otrs-3.0.11/Kernel/Config.pm.dist	2011-10-11 14:01:57.000000000 +0200
+++ otrs-3.0.11.new//Kernel/Config.pm.dist	2011-11-04 09:11:31.300206326 +0100
@@ -59,7 +59,7 @@
     # ---------------------------------------------------- #
     # fs root directory
     # ---------------------------------------------------- #
-    $Self->{Home} = '/opt/otrs';
+    $Self->{Home} = '/usr/share/otrs';
 
     # ---------------------------------------------------- #
     # insert your own config settings "here"               #
diff -ur otrs-3.0.11/scripts/apache2-httpd.include.conf otrs-3.0.11.new//scripts/apache2-httpd.include.conf
--- otrs-3.0.11/scripts/apache2-httpd.include.conf	2011-10-11 14:01:58.000000000 +0200
+++ otrs-3.0.11.new//scripts/apache2-httpd.include.conf	2011-11-04 09:13:04.620205082 +0100
@@ -4,8 +4,8 @@
 # --
 
 # agent, admin and customer frontend
-ScriptAlias /otrs/ "/opt/otrs/bin/cgi-bin/"
-Alias /otrs-web/ "/opt/otrs/var/httpd/htdocs/"
+ScriptAlias /otrs/ "/usr/share/otrs/bin/cgi-bin/"
+Alias /otrs-web/ "/usr/share/otrs/var/httpd/htdocs/"
 
 # activate this if you are using an Oracle database
 #SetEnv ORACLE_HOME /path/to/your/oracle/
@@ -17,7 +17,7 @@
 <IfModule mod_perl.c>
 
     # load all otrs modules
-    Perlrequire /opt/otrs/scripts/apache2-perl-startup.pl
+    Perlrequire /usr/share/otrs/scripts/apache2-perl-startup
 
     # Apache::Reload - Reload Perl Modules when Changed on Disk
     PerlModule Apache2::Reload
@@ -40,26 +40,26 @@
 </IfModule>
 
 # directory settings
-<Directory "/opt/otrs/bin/cgi-bin/">
+<Directory "/usr/share/otrs/bin/cgi-bin/">
     AllowOverride None
     Options +ExecCGI -Includes
     Order allow,deny
     Allow from all
 </Directory>
-<Directory "/opt/otrs/var/httpd/htdocs/">
+<Directory "/usr/share/otrs/var/httpd/htdocs/">
     AllowOverride None
     Order allow,deny
     Allow from all
 </Directory>
 
 <IfModule mod_headers.c>
-    <Directory "/opt/otrs/var/httpd/htdocs/skins/*/*/css-cache">
+    <Directory "/usr/share/otrs/var/httpd/htdocs/skins/*/*/css-cache">
         <FilesMatch "\.(css|CSS)$">
             Header set Cache-Control "max-age=2592000 must-revalidate"
         </FilesMatch>
     </Directory>
 
-    <Directory "/opt/otrs/var/httpd/htdocs/js/js-cache">
+    <Directory "/usr/share/otrs/var/httpd/htdocs/js/js-cache">
         <FilesMatch "\.(js|JS)$">
             Header set Cache-Control "max-age=2592000 must-revalidate"
         </FilesMatch>
diff -ur otrs-3.0.11/scripts/apache2-perl-startup.pl otrs-3.0.11.new//scripts/apache2-perl-startup.pl
--- otrs-3.0.11/scripts/apache2-perl-startup.pl	2011-10-11 14:01:58.000000000 +0200
+++ otrs-3.0.11.new//scripts/apache2-perl-startup.pl	2011-11-04 09:11:31.300206326 +0100
@@ -35,9 +35,9 @@
 use ModPerl::Util;
 
 # set otrs lib path!
-use lib "/opt/otrs/";
-use lib "/opt/otrs/Kernel/cpan-lib";
-use lib "/opt/otrs/Custom";
+use lib "/usr/share/otrs/";
+use lib "/usr/share/otrs/Kernel/cpan-lib";
+use lib "/usr/share/otrs/Custom";
 
 # pull in things we will use in most requests so it is read and compiled
 # exactly once
--- otrs-3.0.11/scripts/redhat-rcotrs-config	2011-10-11 14:01:57.000000000 +0200
+++ otrs-3.0.11.new/scripts/redhat-rcotrs-config	2011-12-20 14:37:10.646969824 +0100
@@ -12,7 +12,7 @@
 # 
 # OTRS home (root) directory
 #
-OTRS_ROOT=/opt/otrs
+OTRS_ROOT=/usr/share/otrs
 
 # if one of this is false, it will not be checked at the startup!
 # Note: (may be the database isn't on the same host! --> DB_RUNNING=0) 
